examples/java/pom.xml

```diff
@@ -1,29 +1,31 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>com.example</groupId>
+  <groupId>io.micronaut.maven.examples</groupId>
   <artifactId>demo</artifactId>
   <version>0.1</version>
   <packaging>${packaging}</packaging>
 
   <parent>
     <groupId>io.micronaut</groupId>
     <artifactId>micronaut-parent</artifactId>
-    <version>3.7.5-SNAPSHOT</version>
+    <version>4.0.0-SNAPSHOT</version>
   </parent>
 
   <properties>
-    <jdk.version>8</jdk.version>
-    <release.version>8</release.version>
-    <micronaut.version>3.7.4-SNAPSHOT</micronaut.version>
-    <micronaut-maven-plugin.version>3.5.1-SNAPSHOT</micronaut-maven-plugin.version>
-    <exec.mainClass>io.micronaut.build.examples.Application</exec.mainClass>
+    <jdk.version>17</jdk.version>
+    <release.version>17</release.version>
+    <micronaut.version>4.0.0-SNAPSHOT</micronaut.version>
+    <micronaut-maven-plugin.version>4.0.0-SNAPSHOT</micronaut-maven-plugin.version>
```

Comment:
:x: There is something fishy here, since the `parent` module declares that version too, and it's using an old version. Either the Maven plugin version should be declared here, or in the parent. I reckon there's a chicken and egg problem, though.

See view-source:https://s01.oss.sonatype.org/content/repositories/snapshots/io/micronaut/micronaut-parent/4.0.0-SNAPSHOT/micronaut-parent-4.0.0-20230214.115349-152.pom and https://github.com/micronaut-projects/micronaut-platform/blob/840460a7851bbdfb602eafd8d6a0117a2e495526/gradle/libs.versions.toml#L94-L93

Reply:
I see, thanks for the clarification!